rbd-target-api.py

```diff
@@ -1670,6 +1670,26 @@ def _targetinfo(target_iqn):
     }), 200
 
 
+@app.route('/api/gatewayinfo', methods=['GET'])
+@requires_restricted_auth
+def gatewayinfo():
+    """
+    Returns the number of active sessions on local gateway
+    **RESTRICTED**
+    Examples:
+    curl --insecure --user admin:admin -X GET
+        http://192.168.122.69:5000/api/gatewayinfo
+    """
+    local_gw = this_host()
+    num_sessions = 0
+    for target_iqn, target in config.config['targets'].items():
+        if local_gw in target['portals']:
```

Comment:
Nit: should it return an error if the GW isn't in the portals list? 404 or 400?

Reply:
@dillaman I've added a validation that checks if `local_gw` is in `config['gateways']`